cmd/create/machinepool/cmd_test.go

```diff
@@ -0,0 +1,115 @@
+package machinepool
+
+import (
+	"go.uber.org/mock/gomock"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/machinepool"
+	mpOpts "github.com/openshift/rosa/pkg/options/machinepool"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/openshift/rosa/pkg/test"
+)
+
+var _ = Describe("Create machine pool", func() {
```

Comment:
@den-rgb I love the conciseness and readability of these tests. Nice work!

Reply:
Thank you